src/lib/runner_test.rs

```diff
@@ -961,6 +963,96 @@ fn run_task_failed_condition_script_doesnt_change_env() {
     assert_eq!(std::env::var(NEW_ENV_VAR), Err(VarError::NotPresent));
 }
 
+#[test]
+#[ignore]
+fn run_task_rust_script_with_args_and_rust_condition_script_with_args() {
+    let dummy_path = "dummy/path";
+
+    // Create temporary directory to store outputs of condition script and script
+    let output_dir = tempdir().unwrap();
+    let condition_script_output_file = output_dir.path().join("condition-script-output.txt");
+    let script_output_file = output_dir.path().join("script-output.txt");
+
+    let flow_info = FlowInfo {
+        config: Config::default(),
+        task: "test".to_string(),
+        env_info: EnvInfo {
+            rust_info: RustInfo::new(),
+            crate_info: CrateInfo::new(),
+            git_info: GitInfo::new(),
+            ci_info: ci_info::get(),
+        },
+        disable_workspace: false,
+        disable_on_error: false,
+        allow_private: false,
+        skip_init_end_tasks: false,
+        skip_tasks_pattern: None,
+        cli_arguments: None,
+    };
+
+    let step = Step {
+        name: "test".to_string(),
+        config: Task {
+            install_crate: Some(InstallCrate::CrateInfo(InstallCrateInfo {
```

Comment:
`install_crate` here doesnt do what i intended here since its run after the condition script is validated. I should just remove it.

Reply:
@sagiegurari is there a better way to ensure that rust-script 0.35.0 is available for this test?

Reply:
you can create multiple tasks with dependencies.
so one task will do install and another task depending on the first, will do the condition.